pkg/flare/clusteragent/cluster_checks.go

```diff
@@ -75,7 +75,7 @@ func GetClusterChecks(w io.Writer, checkName string) error {
 	if len(cr.Dangling) > 0 {
 		fmt.Fprintf(w, "=== %s configurations ===\n", color.RedString("Unassigned"))
 		for _, c := range cr.Dangling {
-			flare.PrintConfig(w, c, checkName)
+			flare.PrintClusterCheckConfig(w, c, checkName)
```

Comment:
@clamoriniere For this PR I decided to leave the cluster check `v1` untouched. We currently do not scrub the integration config information, which is something that we need to do, but it would require more changes. This PR is already big enough, that I think is best to limit the size. 

Once we merge this one and validate everything works as expected we can open a new PR that focus on scrubbing the integration config and precompute the check ID for the cluster check

Reply:
Sounds good